src/ch/jamiete/hilda/Hilda.java

```diff
@@ -168,11 +145,13 @@ public void start() {
         this.plugins.enablePlugins();
         Hilda.getLogger().info("Plugins enabled!");
 
+        Hilda.getLogger().info("Connecting to database...");
+        this.db.connect();
```

Comment:
There needs to be a connected log

Reply:
There is in the DB class.

Reply:
It should be in this class as is convention.

Reply:
But should it fail to connect due to a SQLException we do not want it to log connected, as a result it should be logged in the DB class. As in convention, see your plugins.PluginManager class.